README.md

```diff
@@ -1 +1,20 @@
 # hw01
+# Introduction
+My name is Yilin Zhu, a graduate student from the MAPSS program majoring in sociology. This year I just finished my undergraduate education in sociology at Sun Yat-sen University.
+
+Here is a graduation photo taken with my best friend.
+<a href="https://sm.ms/image/k8R4mn1eZwzF5S7" target="_blank"><img src="https://i.loli.net/2020/10/04/k8R4mn1eZwzF5S7.jpg" ></a>
```

Comment:
The <a href> </a> part is a HTML code. The markdown syntax to attach image is something like ![](). For example, please check if `![myphoto](https://i.loli.net/2020/10/04/k8R4mn1eZwzF5S7.jpg)` works for you.

Reply:
# #Thank you. It works for me. I also wonder if there are any other methods for inserting the photo, for example, uploading the photo to GitHub. I am a little confused that why HTML code does not work here since the photo also appears on my page before, which makes me feel like it somehow works. @nerdizzyz

Reply:
@Yilin-zhu Oh, you can upload a photo file to your GitHub repo, get the URL address, and render it with the markdown. You can upload the photo file with the R studio using the same way you commit and push the README.md file by putting the photo file to the local folder in your system and synchronizing it with the remote repo. Please let me know this explanation is unclear to you.

Reply:
That makes sense to me. Thank you!